libs/platform/experience/src/mixins/content.mixin.ts

```diff
@@ -21,10 +21,24 @@ export declare class ContentMixinInterface<OptionsType, ContentType>
 
   protected data: { options: OptionsType; content: ContentType };
 
+  /**
+   * @deprecated
```

Comment:
I assume we want to drop those soon? (when finishing refactor I guess) As ideally we won't have deprecared in 1.0.

Reply:
For sure. I just want a mechanism during the refactors to highlight these thigns